apps/Gruntfile.js

```diff
@@ -625,6 +625,7 @@ describe('entry tests', () => {
   };
 
   var otherEntries = {
+    header: './src/sites/studio/pages/header.js',
```

Comment:
please move this to `codeStudioEntries`. it will be better for webpack bundling performance, since then ScriptTeacherPanel and such will get pulled out into a common chunk